scripts/maintenance/computational-clusters/autoscaled_monitor/core.py

```diff
@@ -10,9 +10,10 @@
 import parse
 import rich
 import typer
+from common_library.pydantic_type_adapters import ByteSizeAdapter
```

Comment:
can you please revert all the changes in here?
This is a standalone package that does not need nore want to depend on anything.

Reply:
ohh boy missed this one. thanks